cmd/cli.go

```diff
@@ -72,7 +72,7 @@ func main() {
 	if token, ok := os.LookupEnv(envToken); ok {
 		client.SetAccessToken(token)
 	} else {
-		fatal("Missing access token")
+		panic("Missing SLOPPY_APITOKEN, please login by exporting your token https://admin.sloppy.io/account/tokens")
 	}
```

Comment:
Should still stay the `fatal` call.

Reply:
when I switch to `fatal` I get extra `%!(EXTRA []interface {}=[])` string in error message
`Missing SLOPPY_APITOKEN, please login by exporting your token https://admin.sloppy.io/account/tokens%!(EXTRA []interface {}=[])`
is there a way to avoid that?

Reply:
Yes, since fatal accepts a "string format" string in first place there have to be such a replacement token like `%s`.

`fatal` is just a pass through to `printError` with exit 1 afterwards.

You can add a check in `printError` if len(v) == 0 then just use fmt.Sprint instead of Sprintf

Reply:
@malud we can't get this working either - can you please give it a try later, once you've got internet access again